frontend/src/Components/CreateCommentComponent.tsx

```diff
@@ -240,6 +241,27 @@ export default function CreateCommentComponent(props: CreateCommentProps) {
         suggestResults.style.setProperty('left', left.toString() + 'px');
     });
 
+    useEffect(() => {
+        document.addEventListener('paste', handlePaste);
```

Comment:
What happens if multiple comment fields are expanded?

Maybe we should just use `onPaste={handlePaste}` in textarea?